css/styles.css

```diff
@@ -0,0 +1,64 @@
+body {
+	
+	background-color: #9BB4CE;
+	color: black;
+	text-align: center;
+	font-family: helvetica;
+	font-family: 'Montserrat', sans-serif;
```

Comment:
:hammer: You have two `font-family` declarations in a row. Remember that in CSS, lower declarations take precedence, so the above declaration should just be removed.